catalyst_voices/lib/widgets/common/link_text.dart

```diff
@@ -41,13 +41,16 @@ class LinkText extends StatelessWidget {
       decorationStyle: underline ? TextDecorationStyle.solid : null,
     );
 
-    return GestureDetector(
-      onTap: onTap,
-      child: DefaultTextStyle.merge(
-        style: style,
-        child: Text(
-          data,
-          style: effectiveStyle,
+    return MouseRegion(
```

Comment:
@damian-molinski the lack of pointer on `LinkText` was a bug, not a requested feature, right?

Reply:
Correct, didn't think about it